vispy/visuals/image.py

```diff
@@ -89,17 +89,10 @@
 
 _apply_clim_float = """
     float apply_clim(float data) {
-        if ($clim.x < $clim.y) {{
-            data = clamp(data, $clim.x, $clim.y);
-        }} else if ($clim.x > $clim.y) {{
-            data = clamp(data, $clim.y, $clim.x);
-        }} else {{
-            // clims are the same, show minimum colormap value
-            return 0.0;
-        }}
-        data = data - $clim.x;
-        data = data / ($clim.y - $clim.x);
-        return max(data, 0);
+
+        data = clamp(data, $clim.x, $clim.y);
```

Comment:
How does this work for flipped color limits? For example, a floating point texture with values between `0.0` and `100.0` and I specify `clim=(80.0, 20.0)` because I want the colormap flipped?

Reply:
Good point, I had overlooked this use-case. Although one could argue that you should use a reversed colormap in this case, let's revive this feature :)

Reply:
From a GUI point of view though and for performance, this is much faster. Flipping the colormap requires manipulating the colormap instance assuming there is an array of colors to flip. Some colormaps are not an array of colors and are actual functions that get called so a simple inverse is hard to do. Even for arrays of colors, that means uploading new color data to the texture-backed colormap.